README.md

```diff
@@ -1,31 +1,37 @@
-# JekyllPagesApi [![Build Status](https://travis-ci.org/18F/jekyll_pages_api.svg?branch=master)](https://travis-ci.org/18F/jekyll_pages_api)
+# Jekyll Pages API [![Build Status](https://travis-ci.org/18F/jekyll_pages_api.svg?branch=master)](https://travis-ci.org/18F/jekyll_pages_api)
 
-TODO: Write a gem description
+Jekyll Pages API is a [Jekyll](http://jekyllrb.com) [Plugin](http://jekyllrb.com/docs/plugins/) that generates a JSON file with data for all the Pages in your Site.
 
-## Installation
+## Usage
 
 Add this line to your application's Gemfile:
 
 ```ruby
-gem 'jekyll_pages_api'
+group :jekyll_plugins do
+  gem 'jekyll_pages_api', git: 'https://github.com/18F/jekyll_pages_api.git'
+end
```

Comment:
Was going to wait to get the instructions in there before publishing - I'll do a follow-up PR.

/cc https://github.com/18F/jekyll_pages_api/issues/8